### DISCUSSION TITLE
Random idea regarding `main`

### DISCUSSION THREAD

**Discussion by llllvvuu** [nvim-treesitter/nvim-treesitter#5401] [General]
I looked at the README.md for `main` and it seems like so many things were upstreamed into Neovim that this plugin is more similar to https://github.com/neovim/nvim-lspconfig or https://github.com/williamboman/mason-lspconfig.nvim

I wonder if a rebrand to `nvim-treesitter-config` would make sense? I can't imagine it'd be much harder than transitioning people within the `nvim-treesitter` namespace with the whole semver/deprecation song-and-dance; it also clarifies that the plugin isn't a requirement to use treesitter

(Total spitball that I thought about for 30 seconds at most)

**Comment by clason** [comment]
Thank you for the suggestion, but that would just add unnecessary churn on top of already difficult to manage churn.

**Reply by llllvvuu** [reply]
Fair enough. I was thinking it would be similar since basically all of `master` will already need a `@deprecate` / `vim.deprecate`, but maybe there's a more graceful way to kick plugin authors off of the current `master` APIs

**Reply by clason** [reply]
Well, we still have to support downstream plugins for a (long) while. Renaming breaks both branches, and a new repo will make sharing query updates needlessly difficult.
